mestopy/mestopy.py

```diff
@@ -1 +1,242 @@
-"""Main module."""
+import numpy as np
+from pyfar import Signal
+
+
+# Class to generate ref-Objects, that can bei part of the MeasurementChain
+class DeviceObj(object):
+    """Class for device in MeasurementChain.
+
+    This class holds methods and properties of a device in the
+    'MeasurementChain' class.
+    """
+
+    def __init__(self, data, sens=1, name=''):
+        """Init DeviceObj with data.
+
+        Attributes
+        ----------
+        data : Signal
+            Signal with the inverted frequency response of the device.
+        sens : float
+            Sensitivity of the device as a factor.
+        name : str
+            Name of the device.
+        """
+        self.sens = sens
+        self.name = name
+        self.data = data
+
+    @property
+    def freq(self):
+        """Return the frequency responses of the device, multiplied by
+        the sensitivity.
+        """
+        return self.data * self.sens
```

Comment:
How about this
```suggestion
        if self.data is not None:
            return self.data * self.sens
        else:
            return self.sens
```

If we do this the doctring needs an update to say that it either returns a signal or a scalar